.circleci/config.yml

```diff
@@ -349,7 +349,7 @@ jobs:
 
   create-zip:
     docker:
-      - image: mhart/alpine-node:14
+      - image: mhart/alpine-node:16
```

Comment:
Let's change that to `cimg/node:16.19` as we did in the plugin

Reply:
Can you also remove the lines that use `apk` ([this](https://github.com/greenpeace/planet4-master-theme/blob/162891026dc6a4c4aae51ef1fc44de00e96afdd9/.circleci/config.yml#L356), [this](https://github.com/greenpeace/planet4-master-theme/blob/162891026dc6a4c4aae51ef1fc44de00e96afdd9/.circleci/config.yml#L375) and [this](https://github.com/greenpeace/planet4-master-theme/blob/162891026dc6a4c4aae51ef1fc44de00e96afdd9/.circleci/config.yml#L407)), since the `cimg` images are not alpine based and there is no `apk` command. The packages we install in these lines seem to be already included anyway in these images.